services/influxdb/config.go

```diff
@@ -63,6 +63,23 @@ func NewConfig() Config {
 	}
 }
 
+func (c *Config) SetDefaultValues() {
+	if len(c.ExcludedSubscriptions) == 0 {
+		c.ExcludedSubscriptions = map[string][]string{
+			stats.DefaultDatabse: []string{stats.DefaultRetentionPolicy},
+		}
+	}
```

Comment:
@nathanielc Oh ! I think you're right, maybe I have to remove these lines ? (67-71)